src/components/PlaceSearch.vue

```diff
@@ -63,10 +101,22 @@ const getPlaces = async (value) => {
     try {
       const response = await fetch(
         `https://kataster.bev.gv.at/api/all/?term=${encodeURIComponent(value)}`,
-        { signal },
+        { signal }
       );
       const { data } = await response.json();
-      items.value = data?.features || [];
+      const itemValue = data?.features || [];
+
+      items.value = itemValue.sort(
+        /**
+         * @param {placeItem} a
+         * @param {placeItem} b
+         */
+        (a, b) => {
+          return (
+            Number(a.properties.objectType) - Number(b.properties.objectType)
```

Comment:
this would lead to a result-list like this, is this the expected behaviour?


![Screenshot 2024-10-30 at 16 00 46](https://github.com/user-attachments/assets/19299812-7098-4981-a258-f01649be5970)


on a side note, the [object type lookup](https://kataster.bev.gv.at/#/center/13.35,47.77/zoom/7.1/vermv/0.6:~:text=sich%20anhand%20des-,objectTypes,-%5B1.%20Bundesland%2C%202) seems to be wrong, I don't get any results of type "politische Gemeinde" or "Hausnummer"